### ISSUE TITLE
file too large to burn

### ISSUE THREAD

**Issue by Gizmo564** [arduinocelentano/cdimage#7]
The file created was 799 mb when I chose the option for a 700 mb disk, computer would not let me burn it.

![erterrtterertrtertertrteretrtereert](https://user-images.githubusercontent.com/101212391/179081756-badd9b7d-3a27-4b13-825c-7f21d6944368.PNG)

**Comment by cliftbar** [comment]
That looks like its trying to burn a _data_ CD, not an _audio_ CD.  They're different, doing it in audio format can hold the whole file the program makes.  The differences is something around error correction and sector size that I don't understand, but probably data needs more of both.

On windows I couldn't get it to burn as an audio, it kept dropping into windows media player, which complained about the file and wouldn't proceed.  I gave up and used wodim (cdrecord) on a linux vm, but maybe another 3rd party burning software would do it.

**Comment by KirgoPlayer** [comment]
So with wodim have you managed to burn a disc?

**Comment by cliftbar** [comment]
yeah, `wodim -audio dev=... <file>` (from memory), file is 891,169KB big

**Comment by KirgoPlayer** [comment]
> yeah, `wodim -audio dev=/dev/sbd2 <file>` (from memory, substitute your device), file is 891,169KB big (if the question was for me)

Can you send a photo please?

**Comment by cliftbar** [comment]
![PXL_20220716_011027745_exported_1658014359621 - Copy](https://user-images.githubusercontent.com/3311259/179375153-7d00beee-d270-4fb6-962c-7623cdddf8fc.jpg)

this is on verbatim 12x CD-RWs, params tweaked from the SERL 2 option (tl0: 22951.25, dtr: 1.3865961).  Still needs tuning, the vertical line is supposed to be straight.

**Comment by KirgoPlayer** [comment]
> ![PXL_20220716_011027745_exported_1658014359621 - Copy](https://user-images.githubusercontent.com/3311259/179375153-7d00beee-d270-4fb6-962c-7623cdddf8fc.jpg)
> 
> this is on verbatim 12x CD-RWs, params tweaked from the SERL 2 option (tl0: 22951.25, dtr: 1.3865958).  Still needs tuning, the vertical line is supposed to be straight.
> 

Please update me If you discover the geometry of other CD's or perfect the geometry of the 12x CD. Where can I contact you in private?

**Comment by arduinocelentano** [comment]
800Mb is OK for Audio CD. It uses less error correction techniques compared to Data CD.

**Comment by arduinocelentano** [comment]
> ![PXL_20220716_011027745_exported_1658014359621 - Copy](https://user-images.githubusercontent.com/3311259/179375153-7d00beee-d270-4fb6-962c-7623cdddf8fc.jpg)
> 
> this is on verbatim 12x CD-RWs, params tweaked from the SERL 2 option (tl0: 22951.25, dtr: 1.3865958). Still needs tuning, the vertical line is supposed to be straight.

Nice job! In fact if you are able to see radial lines, you are nearly there. Unfortunately, even discs of the same model sometimes have slightly different parameters.
Can I use your photo on a [project's page](https://hackaday.io/project/186303-burning-pictures-on-a-compact-disc-surface)? It might be helpful for others if you take a picture of its label or box as well. Just in case someone would look for a similar disc to experiment with.

**Comment by KirgoPlayer** [comment]
> > ![PXL_20220716_011027745_exported_1658014359621 - Copy](https://user-images.githubusercontent.com/3311259/179375153-7d00beee-d270-4fb6-962c-7623cdddf8fc.jpg)
> > 
> > this is on verbatim 12x CD-RWs, params tweaked from the SERL 2 option (tl0: 22951.25, dtr: 1.3865958). Still needs tuning, the vertical line is supposed to be straight.
> 
> Nice job! In fact if you are able to see radial lines, you are nearly there. Unfortunately, even discs of the same model sometimes have slightly different parameters.
> Can I use your photo on a [project's page](https://hackaday.io/project/186303-burning-pictures-on-a-compact-disc-surface)? It might be helpful for others if you take a picture of its label or box as well. Just in case someone would look for a similar disc to experiment with.

Maybe any ideas in how to make that line straight? That also happened to me when I tried with Verbatim 12x ![IMG_20220723_165742_310~2.jpg](https://user-images.githubusercontent.com/106423698/180618806-1e4b5708-2af8-4507-b2fc-dca63296666a.jpg)

**Comment by cliftbar** [comment]
@arduinocelentano feel free to use the pic for the project page.  I already tossed the packaging so no pictures of that, but I have the amazon order: discs are `Verbatim CD-RW 700MB 2X-4X DataLifePlus Silver Inkjet Printable with Branded Hub - 50pk Spindle - P/N 95159`,
